packages/components/src/slot-fill/bubbles-virtually/slot-fill-provider.js

```diff
@@ -1,8 +1,14 @@
 // @ts-nocheck
+/**
+ * External dependencies
+ */
+import { ref as valRef } from 'valtio';
+import { proxyMap } from 'valtio/utils';
```

Comment:
Any reason why we want to introduce yet another package here to solve this issue? Is it too tedious to solve in vanilla React?

Reply:
Yes, I found it a bit difficult (also a lot of code) to recreate the same behavior with only React. Basically we'd want multiple event emitters, we want to subscribe to changes for each slot separately and each fills for a slot.

Reply:
Sorry, I accidentally marked this as resolved.